magicctapipe/image/__init__.py

```diff
@@ -9,10 +9,15 @@
     get_leakage,
 )
 
+from .calib import (
+    calibrate,   
+)
+
 __all__ = [
     "MAGICClean",
     "PixelTreatment",
     "get_num_islands_MAGIC",
+    "calibrate",
     "clean_image_params",
-    "get_leakage",
+    "get_leakage",    
```

Comment:
again some spaces were added that marked the line as changed unnecessarily

Reply:
Done